dev_scripts/template_pyxis.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
```

Comment:
In my opinion this script does not really provide much value. I think we should just remove it. What are your thoughts @gnaponie?

Reply:
I guess it was useful for manual testing of the code. If everything is covered by unittesting perhaps we can remove it.

Reply:
I can remove if from git but leave it in my local repo. Just wanted to share, because it helped me a lot when I wanted to query Pyxis for different stuff like indices, images and so on.